scipy/ndimage/tests/test_filters.py

```diff
@@ -2489,7 +2514,8 @@ def check_func_serial(self, n, fun, args, out):
         for i in range(n):
             fun(*args, output=out[i, ...])
 
-    @skip_xp_backends("jax.numpy", reasons=["output array is read-only"])
+    @skip_xp_backends("jax.numpy", reasons=["output array is read-only"],
```

Comment:
All tests are decorated the exact same way, and that is unlikely to change because it's all due to the central utility functions `check_func_*` here which use `output=`. So can this skip be applied once at the class level instead?

Reply:
done